packages/botonic-react/src/utils.js

```diff
@@ -61,3 +61,6 @@ export const _getThemeProperty = theme => (
     }
   }
 }
+
+export const ConditionalWrapper = ({ condition, wrapper, children }) =>
```

Comment:
this is so cool :100:

Reply:
yes it's so cool! Reference: https://blog.hackages.io/conditionally-wrap-an-element-in-react-a8b9a47fab2